www/src/scss/_theme-a-new-day.scss

```diff
@@ -93,6 +103,7 @@ $base-text: #4e4e4e;
       color: $base-text;
     }
 
+    input:focus+div {
```

Comment:
duplicate to the line below this one. delete this one since the bottom is formatted properly.